docs/ref/templates/builtins.txt

```diff
@@ -2430,18 +2431,16 @@ slightly different call:
     {% static "images/hi.jpg" as myphoto %}
     <img src="{{ myphoto }}"></img>
 
-.. note::
 
-    The :mod:`staticfiles<django.contrib.staticfiles>` contrib app also ships
-    with a :ttag:`static template tag<staticfiles-static>` which uses
-    ``staticfiles'`` :setting:`STATICFILES_STORAGE` to build the URL of the
-    given path (rather than simply using :func:`urllib.parse.urljoin` with the
-    :setting:`STATIC_URL` setting and the given path). Use that instead if you
-    have an advanced use case such as :ref:`using a cloud service to serve
-    static files<staticfiles-from-cdn>`::
+If the :mod:`~django.contrib.staticfiles` contrib app is configured,
+the static template tag, will serve all files from
+defined storage by :setting:`STATICFILES_STORAGE`.
+
+.. versionchanged:: 1.10
 
-        {% load static from staticfiles %}
-        <img src="{% static "images/hi.jpg" %}" alt="Hi!" />
+    In older versions, you had to use ``{% load static from staticfiles %}`` in
+    your template to serve files from the storage defined in
+docs/releases/1.10.txt    :setting:`STATICFILES_STORAGE`. This is no longer required.
```

Comment:
Edits for this file weren't applied correctly and the changes in the 1.4 release are needed to prevent a broken link.